templates/books-show.html

```diff
@@ -0,0 +1,39 @@
+<div class="container">
+  <div class="row">
+    <div class="col-md-10 col-md-offset-1">
+      <div class="panel panel-default">
+        <div class="panel-body">
+          <ul class="list-group form-inline">
+            <div class="row">
+              <div class="col-md-3 col-xs-12 thumbnail">
+                <img src="{{booksShowCtrl.book.image}}">
+              </div>
+              <div class="col-md-9 col-xs-12">
+                  <li class="list-group-item">
+                    <h4 class="inline-header">Book Title: {{booksShowCtrl.book.title}}</h4>
+                    <h4 class="inline-header">Author: {{booksShowCtrl.book.author}}</h4>
+                    <h4 class="inline-header">Date Released: {{booksShowCtrl.book.releaseDate}}</h4>
+                  </li>
+              </div>
+              <div class='panel-footer'>
+                <div class='panel-footer'>
+                  <button class='btn btn-danger' ng-click="booksShowCtrl.deleteBook(book)">Delete Book</button>
+                  <button class='btn btn-info' ng-hide="editing" ng-click="editing = true">Edit Book</button>
+                  <button class='btn btn-success' ng-show="editing" ng-click="booksShowCtrl.editBook(book); editing = false">Save Changes</button>
+                </div>
+                <div ng-show="editing">
+                  <form ng-show="editing" ng-submit="booksShowCtrl.editBook(book)">
+                    <input type="text" ng-model="booksShowCtrl.book.title" class="form-control" placeholder="Title">
```

Comment:
You should use labels rather than placeholder text to indicate what each box is for. See https://www.nngroup.com/articles/form-design-placeholders/